src/dbtVersions.js

```diff
@@ -0,0 +1,585 @@
+            "dbtversion": ">=0.18.0 <0.20.0",
+            "packages": {
+                "dbt-labs/dbt_utils": ">=0.6.0 <0.7.0"
+            }
+        },
+        "0.3.1": {
+            "dbtversion": ">=0.18.0 <0.20.0",
+            "packages": {
+                "dbt-labs/dbt_utils": ">=0.6.0 <0.7.0"
+            }
+        },
+        "0.4.0": {
+            "dbtversion": ">=0.18.0 <0.20.0",
+            "packages": {
+                "dbt-labs/dbt_utils": ">=0.6.0 <0.7.0"
+            }
+        },
+        "0.4.1": {
+            "dbtversion": ">=0.18.0 <0.20.0",
+            "packages": {
+                "dbt-labs/dbt_utils": ">=0.6.0 <0.7.0"
+            }
+        },
+        "0.5.0": {
+            "dbtversion": ">=0.20.0 <1.1.0",
+            "packages": {
+                "dbt-labs/dbt_utils": ">=0.7.0 <0.8.0"
+            }
+        },
+        "0.6.0": {
+            "dbtversion": ">=0.20.0 <1.1.0",
+            "packages": {
+                "dbt-labs/dbt_utils": ">=0.7.0 <0.8.0"
+            }
+        },
+        "0.7.0": {
+            "dbtversion": ">=1.0.0 <2.0.0",
+            "packages": {
+                "dbt-labs/dbt_utils": ">=0.8.0 <0.9.0"
+            }
+        },
+        "0.8.0": {
+            "dbtversion": ">=1.0.0 <2.0.0",
+            "packages": {
+                "dbt-labs/dbt_utils": ">=0.8.0 <0.9.0"
+            }
+        },
+        "0.9.0": {
+            "dbtversion": ">=1.0.0 <2.0.0",
+            "packages": {
+                "dbt-labs/dbt_utils": ">=0.8.0 <0.9.0"
+            }
+        },
+        "0.10.0": {
+            "dbtversion": ">=1.0.0 <2.0.0",
+            "packages": {
+                "dbt-labs/dbt_utils": ">=0.8.0 <0.9.0"
+            }
+        },
+        "0.11.0": {
+            "dbtversion": ">=1.0.0 <2.0.0",
+            "packages": {
+                "dbt-labs/dbt_utils": ">=0.8.0 <0.9.0"
+            }
+        },
+        "0.12.0": {
+            "dbtversion": ">=1.0.0 <2.0.0",
+            "packages": {
+                "dbt-labs/dbt_utils": ">=0.8.0 <0.9.0"
+            }
+        },
+        "0.12.1": {
+            "dbtversion": ">=1.0.0 <2.0.0",
+            "packages": {
+                "dbt-labs/dbt_utils": ">=0.8.0 <0.9.0"
+            }
+        },
+        "0.12.2": {
+            "dbtversion": ">=1.0.0 <2.0.0",
+            "packages": {
+                "dbt-labs/dbt_utils": ">=0.8.0 <0.10.0"
+            }
+        },
+        "0.12.3": {
+            "dbtversion": ">=1.0.0 <2.0.0",
+            "packages": {
+                "dbt-labs/dbt_utils": ">=0.8.0 <0.10.0"
+            }
+        },
+        "0.13.0": {
+            "dbtversion": ">=1.3.0 <2.0.0",
+            "packages": {
+                "dbt-labs/dbt_utils": ">=1.0.0 <2.0.0"
+            }
+        },
+        "0.13.1": {
+            "dbtversion": ">=1.3.0 <2.0.0",
+            "packages": {
+                "dbt-labs/dbt_utils": ">=1.0.0 <2.0.0"
+            }
+        },
+        "0.13.2": {
```

Comment:
What will happen when we forget to add a new version here? The previous version will appear as the latest? Or something worse?

Reply:
Yeah so it's only fetching our packages, if we don't update this it just means we will show an older version as the latest. Absolute worst case if dbt releases version 2 and it looks like we don't suport it when some newer versions do. 

I could have the python script run regularly in an action and raise a PR if there are any changes if we want to make sure new versions are 100% caught?